src/components/EReceiptThumbnail.tsx

```diff
@@ -100,7 +108,8 @@ function EReceiptThumbnail({transaction}: EReceiptThumbnailProps) {
                         fill={secondaryColor}
                         additionalStyles={[styles.fullScreen]}
                     />
-                    {MCCIcon ? (
+                    {isReceiptThumbnail && fileExtension && <Text style={[styles.labelStrong, StyleUtils.getTextColorStyle(colors.black)]}>{fileExtension.toUpperCase()}</Text>}
+                    {MCCIcon && !isReceiptThumbnail ? (
```

Comment:
This condition is making the MCC Icon not show on the money request preview. What is the purpose of this check originally?


https://github.com/Expensify/App/assets/25520267/ec917200-4c32-4da1-9ffd-de42fb0b4782

Reply:
What is the purpose of MCIcons?? Are we sure we want it to be displayed along with the label? 
I mean it wasn't displayed before because it was a static receipt preview thumbnails icons we were displaying.
If yes, how should the layout be ?? Currently if we allow it it would look like this.
![image](https://github.com/Expensify/App/assets/38649957/98d4317c-3bdc-4514-a90f-ca88d1cc32b3)

Reply:
>MCCIcon is for EReceipt, I don't think we have smart scanning combined with the receipt. As you notice that in my video I'm not using the smart scanning.

That was my intention originally. 👍 
So you are agreeing with the current status of not showing MCCIcon for receipt eThumbnail??

Reply:
Despite that, in `ReceiptImage` component you combine the check for eReceipt and thumbnail to show the `EReceiptThumbnail`.

```
    if (isEReceipt || isThumbnail) {
        return (
            <View style={style ?? [styles.w100, styles.h100]}>
                <EReceiptThumbnail
                    transactionID={transactionID ?? ''}
                    borderRadius={style?.borderRadius}
                    fileExtension={fileExtension}
                    isReceiptThumbnail
                />
            </View>
        );
    }
```

This will never show the MCCIcon on the eReceipt.

Reply:
U 're right! Updated 👍